frontend/src/components/HomePage/Body/homePageBody.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './homePageBody.module.css'
+
+function HomePageBody () {
+    return (
+        <div className={styles["body-grid"]}>
+            <p className={styles["home-heading"]}>HOME /</p>
+            <h1 className={styles["body-heading"]}>Learn something new today!</h1>
+            <p className={styles["search-bar-heading"]}>WHAT ARE YOU LOOKING FOR</p>
+            <div className={styles["search"]}>
+                <input type="text" placeholder='Keyboard modding'/>
+                <button className={styles["search-button"]}>SEARCH</button>
```

Comment:
add a `handleSearch` or `handleSubmit` function that gets triggered whenever this button is pressed. you don't have to fully implement the submitting logic (sending the request to the backend) but just have it as a placeholder function for future reference when you do implement it!